tftp/bootstrap.py

```diff
@@ -276,18 +278,13 @@ def startProtocol(self):
             bytes = OACKDatagram(self.resultant_options).to_wire()
         else:
             bytes = ACKDatagram(0).to_wire()
-        self.timeout_watchdog = SequentialCall.run(
-            self.timeout[:-1],
-            callable=self.transport.write, callable_args=[bytes, ],
-            on_timeout=lambda: self._clock.callLater(self.timeout[-1], self.timedOut),
-            run_now=True,
-            _clock=self._clock
-        )
+        self.timeout_watchdog = timedCaller(
+            chain((0,), self.timeout), partial(self.transport.write, bytes),
```

Comment:
I know `bytes` was a previously used variable, but it is also a type in python. I recommend renaming the variable.